jupyterlab_gitplus/handlers.py

```diff
@@ -142,7 +148,12 @@ def post(self):
             if old_commit != new_commit:
                 owner_login, repo_name = get_owner_login_and_repo_name(repo)
                 github_url = GITHUB_ENDPOINT + owner_login + '/' + repo_name + '/commit/' + new_commit
-                reviewnb_url = REVIEWNB_ENDPOINT + owner_login + '/' + repo_name + '/commit/' + new_commit
```

Comment:
(after following my top comment above) we can just use `reviewnb_endpoint`  in place of `REVIEWNB_ENDPOINT` here.

Reply:
Done